terraform-ecs-fargate-service/variables.tf

```diff
@@ -0,0 +1,17 @@
+
+variable "env" {}
+variable "ecs_cluster_id" {}
+variable "vpc_id" {}
+variable "service_port" {}
+variable "service_name" {}
+variable "task_execution_role_arn" {}
+variable "task_role_arn" {}
+variable "load_balancer_listener_arn" {}
+variable "load_balancer_listener_host_header" {}
+variable "load_balancer_listener_host_header_2" {}
```

Comment:
a tohleeee bych mozna spis trochu predelal

Reply:
muzeme o tom zavolat a vysvetlim proc to tam vubec je

Reply:
Nó, můžeme. Zrovna na to koukám. 🙂